usaspending_api/common/management/commands/run_sql.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
```

Comment:
Did we intend to commit this file this time? I know we've been leaving it out so far.

Reply:
Yes this was indeed intentional